apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/resources/schemas/Sgv2IndexesResourceImpl.java

```diff
@@ -41,8 +44,11 @@ public Response getAllIndexesForTable(String keyspaceName, String tableName) {
     }
 
     // check that we're authorized for the table
-    bridge.authorizeSchemaRead(
-        SchemaReads.table(keyspaceName, tableName, Schema.SchemaRead.SourceApi.REST));
+    Uni<List<Boolean>> uni =
+        authorizeSchemaReadsAsync(
+            Arrays.asList(
+                SchemaReads.table(keyspaceName, tableName, Schema.SchemaRead.SourceApi.REST)));
+    Futures.getUninterruptibly(uni.subscribeAsCompletionStage());
```

Comment:
I believe we should check the result of this uni? (i.e. check all the booleans in the list)?

Reply:
Yes, that sounds like something that should be done; will change.

Reply:
And also, that should be chained with the next call (fetchRowsAsync) I think.

Reply:
Was able to change code to do the same in right way, but Ivan pointed out this is odd usage -- no other methods check it explicitly. So will change to use one of other methods on validating access to table.